src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/SlingInitialContentBundleEntryMetaDataCollector.java

```diff
@@ -68,6 +80,20 @@ class SlingInitialContentBundleEntryMetaDataCollector {
         this.contentPackage2FeatureModelConverter = contentPackage2FeatureModelConverter;
         this.newBundleFile = newBundleFile;
         this.jarFile = context.getJarFile();
+        
+        this.defaultContentXmlFile = new File(this.basePath, "defaultContentXml.xml");
```

Comment:
i don't remember having ever seen the name `defaultContentXml` in the context of content packages. is this related to sling-initial content?
if not and this is something you just need here, i would be great if you could add a comment elaborating what this is used for.

Reply:
below you use the string 'default-content.xml'

Reply:
We could rename it.. basically its .content.xml file with sling:Folder as jcr:primaryType. We put it in sling initial content if there is no specification.

Reply:
and .content.xml does not work? or what is the reason?
in either case: i would recommend to then add some comment into the xml file indicating that this was injected by the converter.